rocketpy/Rocket.py

```diff
@@ -960,8 +970,37 @@ def allInfo(self):
             + "{:.3f}".format(self.centerOfMass(0))
             + " m"
         )
-        print("\nAerodynamic Components Parameters")
-        print("Currently not implemented.")
+
+        # Print Aerodynamic Components Information
+        print("\nAerodynamic Components")
+        print(self.aerodynamicSurfaces)
```

Comment:
I would suggest a minor change here so the user can better visualize the outputs:

```suggestion
        for surface in self.aerodynamicSurfaces:
            if 'name' in surface:
                print(surface['name']) 
            else:
                print('Unnamed Surface:')
            print(surface)
```